tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -1061,13 +1057,13 @@
     <value>More</value>
     <comment>Label for a link that navigates the user to a location to learn more about WER local dumps</comment>
   </data>
-  <data name="RunElevatedButtonForWER.Text" xml:space="preserve">
+    <data name="RunElevatedButton.Text" xml:space="preserve">
     <value>Change</value>
-    <comment>The contents of a button to allow the user to trigger elevation to change WER settings</comment>
+    <comment>The contents of a button to allow the user to trigger elevation</comment>
   </data>
-  <data name="RunElevatedButtonForWER.ToolTipService.ToolTip" xml:space="preserve">
-    <value>Project Ironsides needs to run as Admin in order to modify WER collection for an app.</value>
-    <comment>{Locked="Ironsides"} Tooltip for a button that relaunches Project Ironsides as admin to change WER settings</comment>
+    <data name="RunElevatedButton.ToolTipService.ToolTip" xml:space="preserve">
+    <value>Project Ironsides needs to run as Admin in order to get more data for an app.</value>

```

Comment:
Feels like maybe you need multiple strings here?

In this case, we lost the WER-specific tooltip language (get more data for an app is not why we'd want to elevate for WER).

Also, having the button text say "Change" for the Modules page seems weird, as you're not changing anything.